bundle/config/mutator/load_git_details.go

```diff
@@ -21,50 +20,41 @@ func (m *loadGitDetails) Name() string {
 }
 
 func (m *loadGitDetails) Apply(ctx context.Context, b *bundle.Bundle) diag.Diagnostics {
-	// Load relevant git repository
-	repo, err := git.NewRepository(b.BundleRoot)
+	info, err := git.FetchRepositoryInfo(ctx, b.BundleRoot, b.WorkspaceClient())
```

Comment:
Errors used to be logged and not returned to the caller (the `log.Warnf` calls).

Are there cases where this will regress?

Reply:
Updated the logic to match previous warnings + added more clarity about error handling (all errors are now logged).